src/test/s3/util/fileViewerManager.test.ts

```diff
@@ -166,11 +166,27 @@ describe('FileViewerManager', function () {
         await vscode.commands.executeCommand('workbench.action.closeActiveEditor')
     }
 
-    async function getActiveEditor(): Promise<vscode.TextEditor | never> {
-        const editor = await waitUntil(async () => vscode.window.activeTextEditor, { interval: 5 })
-        assert.ok(editor, 'No active text editor found')
+    /**
+     * Waits for _any_ active text editor to appear and have the desired contents.
+     * This is important since their may be delays between showing a new document and
+     * updates to the `activeTextEditor` field.
+     */
+    async function assertTextEditorContains(contents: string): Promise<void | never> {
```

Comment:
Do you plan to move this into testUtil.ts ?

Reply:
Yeah I can move it.